ansible/roles/ocp4_workload_rhacm/tasks/pre_workload.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Implement your Pre Workload deployment tasks here
+- set_fact:
+    ansible_python_interpreter: "/bin/python"
+
+- name: Setup pre-requisite python3 packages
```

Comment:
I'm not quite sure I understand why you can't use the default k8s virtualenv that's already there for this.

Reply:
Changed.